lib/multithreading_lock/Kconfig

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config MULTITHREADING_LOCK
+	bool "Multithreading Lock"
```

Comment:
This needs more help text.

Reply:
Yes, it should be hidden I think. So I'll remove the prompt. But do you want me to add a help text? Or is it ok without?

Reply:
A one sentence help text would be good to have.